extra-dev/packages/rocq-mathcomp-ssreflect/rocq-mathcomp-ssreflect.dev/opam

```diff
@@ -10,45 +10,13 @@ license: "CECILL-B"
 build: [ make "-C" "ssreflect" "-j" "%{jobs}%" ]
 install: [ make "-C" "ssreflect" "install" ]
```

Comment:
Does it actually build and install something?

Reply:
The `all_ssreflect.v` compat file: https://github.com/math-comp/math-comp/blob/master/ssreflect/all_ssreflect.v

Reply:
It should at least build a `all_ssreflect.vo` file and install it...

Reply:
Then it might be worth keeping an entry `"logpath:mathcomp.ssreflect"` to show where it gets installed.